### DISCUSSION TITLE
Invalid plot (RuntimeError: badbit or failbit)

### DISCUSSION THREAD

**Discussion by maxrozenblum2020** [Chia-Network/chia-blockchain#6579] [Support]
Hi I am running on Ubuntu, and am solo (no Hpool, chia-core, etc...). When running `chia plots check` ~15 of my 1000 plots fail to open with the printout:

"badbit or failbit after reading size 4 at position 107539976848 Traceback (most recent call last):                                                                                                                   File "/home/max/chia-blockchain/chia/plotting/plot_tools.py", line 189, in process_file                                                                                                                             prover = DiskProver(str(filename))                                                                                                                                                                            RuntimeError: badbit or failbit after reading size 4 at position 107539976848"

Any insight would be appreciated. 
(1) What causes this (why did it happen to some and not others)? There is no trend in the HDD or tempdrive used to plot these invalid plots.
(2) Are these plots not farmable (some people have told me not to delete until I can confirm this)? If they are not farmable and there is nothing I can do about it, then it just makes sense to delete them and replot.

**Comment by XIII39** [comment]
In my personal experience I have seen this when plots don't fully copy from the tmp drive to the dst drive.. if you look at the file size it might be 102.X GB as opposed to the 108.X GB for k=32 plot..

Usually it happens when you reach the end of the drive storage.
I delete and re-plot

**Comment by BrandtH22** [comment]
@maxrozenblum2020 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
